cluster-provision/k8s/provision.sh

```diff
@@ -15,8 +16,18 @@ export base
 
 cd $DIR
 
-export KUBEVIRT_CGROUPV2="${CGROUPV2}"
+make -C ../gocli cli
```

Comment:
Would it be possible to use an existing gocli image instead of building here? Not sure if we need to add an extra build here.

Reply:
we always build it here already and we must, (see few lines below)
it takes a moment and must be build as part of the provision as it use `provision` command of gocli and the pman command, so if there are changes we need to take them into account for both commands

Reply:
Ok sorry I missed that. Looks good to me.